README.md

```diff
@@ -4,6 +4,19 @@ the Jupyter Software Steering Council.
 
 This repository is **NOT** for opening or discussing JEPs. This can be done in the [dedicated repository](https://github.com/jupyter/enhancement-proposals).
 
+## How the Software Steering Council operates
+
+The Software Steering Council (SSC) is an asynchronous-first council. We do our best not to _require_ synchronous meetings with the whole council since we span various timezones and aim to keep a robust record of all communications. To facilitate this style, we organize our work and track in-flight tasks, using a [Github project board](https://github.com/orgs/jupyter/projects/10/views/1).
```

Comment:
```suggestion
The Software Steering Council (SSC) is an asynchronous-first council. We do our best not to _require_ synchronous meetings with the whole council since we span various timezones and aim to keep a robust record of all communications. To facilitate this style, we organize our work and track in-flight tasks using a [Github project board](https://github.com/orgs/jupyter/projects/10/views/1).
```